content/articles/payment-methods.markdown

```diff
@@ -0,0 +1,34 @@
+---
+title: Payment methods
+excerpt: Payment methods supported by DNSimple.
+categories:
+- DNSimple
+---
+
+# Payment Methods
+
+DNSimple only supports two payment methods:
+
+## Credit card
+
+<note>
+DNSimple doesn't store credit card information. We use a PCI-compliant billing provider to process and store all payment data.
```

Comment:
Sidenote: I recently learned that especially with Chargify we also would need a certification to process the CC and fwd them to Chargify.

As we are switching to Stripe where this is not the case anymore. I do not see any reason to change this.

Reply:
✅